tools/run_tests/performance/scenario_config.py

```diff
@@ -252,6 +252,19 @@ def scenarios(self):
             secure=False,
             categories=[SMOKETEST] + [INPROC] + [SCALABLE])
 
+        yield _ping_pong_scenario(
+            'cpp_protobuf_callback_unary_1channel_100rpcs_1MB',
```

Comment:
Oh, I see you're sending 1MB messages here. Is that intended? Do we want to have a scenario for 1 channel 1 byte as well?

Reply:
Not really, just aped the unit test for async. I''ll add a 1 byte 1 channel scenario. I do have a 1 byte 1 channel scenario for the benchmark run. Though that code sits in google3 and I'll send that for review once these changes are approved and integrated

Reply:
No need to add it here if async doesn't do it.